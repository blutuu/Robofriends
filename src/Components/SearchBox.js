import React from 'react';

const SearchBox = ({searchChange}) => {
  return (
      <>
        <input className="db ml-auto mr-auto mt3 mb4 pa2"
               placeholder="Search Robots"
               type="search"
               onChange={searchChange}/>
      </>
  );
}

export default SearchBox;