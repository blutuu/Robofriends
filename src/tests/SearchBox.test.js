import React from "react";
import { shallow } from "enzyme";
import SearchBox from "../Components/SearchBox";

let wrapper;

describe("SearchBox component testing", () => {
  it("renders SearchBox correctly", () => {
    wrapper = shallow(<SearchBox searchChange="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
