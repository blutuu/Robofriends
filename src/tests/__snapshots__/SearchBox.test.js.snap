// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox component testing renders SearchBox correctly 1`] = `
<Fragment>
  <input
    className="db ml-auto mr-auto mt3 mb4 pa2"
    onChange=""
    placeholder="Search Robots"
    type="search"
  />
</Fragment>
`;
